web/web.go

```diff
@@ -425,6 +425,7 @@ func tmplFuncs(consolesPath string, opts *Options) template_text.FuncMap {
 		},
 		"consolesPath": func() string { return consolesPath },
 		"pathPrefix":   func() string { return opts.ExternalURL.Path },
+		"buildVersion": func() string { return opts.Version.Version },
```

Comment:
`Version.Revision` (or the first 7 characters of that) would be even better for people not building off a release version.